azurerm/internal/services/network/tests/virtual_hub_connection_resource_test.go

```diff
@@ -225,94 +211,90 @@ func testAccAzureRMVirtualHubConnection_complete(data acceptance.TestData) strin
 	template := testAccAzureRMVirtualHubConnection_template(data)
 	return fmt.Sprintf(`
 %s
-
 resource "azurerm_virtual_network" "test2" {
   name                = "acctestvirtnet2%d"
   address_space       = ["10.6.0.0/16"]
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
 }
-
 resource "azurerm_network_security_group" "test2" {
   name                = "acctestnsg2%d"
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
 }
-
 resource "azurerm_subnet" "test2" {
   name                 = "acctestsubnet2%d"
   resource_group_name  = azurerm_resource_group.test.name
   virtual_network_name = azurerm_virtual_network.test2.name
-  address_prefix       = "10.6.1.0/24"
+  address_prefixes     = ["10.6.1.0/24"]
 }
-
 resource "azurerm_subnet_network_security_group_association" "test2" {
   subnet_id                 = azurerm_subnet.test2.id
   network_security_group_id = azurerm_network_security_group.test2.id
 }
-
 resource "azurerm_virtual_hub_connection" "test" {
-  name                                           = "acctestvhubconn-%d"
-  virtual_hub_id                                 = azurerm_virtual_hub.test.id
-  remote_virtual_network_id                      = azurerm_virtual_network.test.id
-  hub_to_vitual_network_traffic_allowed          = true
-  vitual_network_to_hub_gateways_traffic_allowed = false
-  internet_security_enabled                      = false
+  name                      = "acctestvhubconn-%d"
+  virtual_hub_id            = azurerm_virtual_hub.test.id
+  remote_virtual_network_id = azurerm_virtual_network.test.id
+  internet_security_enabled = false
 }
-
```

Comment:
leaving the spacing in here does make this easier to read (since we do link to acceptance tests as examples from time to time) - so it'd be nice to leave these in

Reply:
This is by accident, I copied it from another PR from github web file view (not raw view).
It behaves strangely that the empty line in the range of backtick is stripped, while the other empty lines are reserved 😂